filters.js

```diff
@@ -447,6 +449,40 @@ function operationOnDates(v, arg, operation) {
   }
 }
 
+function toCurrency(currValue, currType) {
+  if (
+    isValidNumber(currValue) &&
+    Math.sign(currValue) === 1 &&
+    _.isString(currType)
+  ) {
+    return { value: currValue, type: currType };
+  }
+  throw new Error("invalid currency value or type");
+}
+
+function toDuration(durValue, durType) {
+  if (
+    isValidNumber(durValue) &&
+    Math.sign(durValue) === 1 &&
+    _.isString(durType)
+  ) {
+    let durationType = durType.toLowerCase();
+
+    switch (durationType) {
+      case "days":
```

Comment:
Let's add an object for this?

`{
  DAYS: "days",
...
}`

Reply:
this is done. added a global object so that we can use the same in the util function for calculating days.